new-pipeline/src/evo.js

```diff
@@ -944,6 +950,8 @@ function saveStateToUrlAndCookie() {
       .is(":checked") ? 1 : 0,
     sanitize: $("input[name=sanitize-toggle]")
       .is(":checked") ? 1 : 0,
+      include_spill: $("input[name=include-spill-toggle]")
```

Comment:
Please back this line out a level of indent (2 spaces) to line up with `sanitize`